stm32-data-gen/src/chips.rs

```diff
@@ -932,9 +934,43 @@ fn process_core(
         peri_kinds.insert(pname, pkind.to_string());
     }
     const GHOST_PERIS: &[&str] = &[
-        "GPIOA", "GPIOB", "GPIOC", "GPIOD", "GPIOE", "GPIOF", "GPIOG", "GPIOH", "GPIOI", "GPIOJ", "GPIOK", "GPIOL",
-        "GPIOM", "GPION", "GPIOO", "GPIOP", "GPIOQ", "GPIOR", "GPIOS", "GPIOT", "DMA1", "DMA2", "BDMA", "DMAMUX",
-        "DMAMUX1", "DMAMUX2", "SBS", "SYSCFG", "EXTI", "FLASH", "DBGMCU", "CRS", "PWR", "AFIO", "BKP", "USBRAM",
+        "GPIOA",
+        "GPIOB",
+        "GPIOC",
+        "GPIOD",
+        "GPIOE",
+        "GPIOF",
+        "GPIOG",
+        "GPIOH",
+        "GPIOI",
+        "GPIOJ",
+        "GPIOK",
+        "GPIOL",
+        "GPIOM",
+        "GPION",
+        "GPIOO",
+        "GPIOP",
+        "GPIOQ",
+        "GPIOR",
+        "GPIOS",
+        "GPIOT",
+        "DMA1",
+        "DMA2",
+        "BDMA",
+        "DMAMUX",
+        "DMAMUX1",
+        "DMAMUX2",
+        "SBS",
+        "SYSCFG",
+        "EXTI",
+        "FLASH",
+        "DBGMCU",
+        "CRS",
+        "PWR",
+        "AFIO",
+        "BKP",
+        "USBRAM",
+        "VREFINTCAL",
```

Comment:
This causes adding `VREFINTCAL` to many more chips, like F7s. is it intentional, have you checked it's correct?

Reply:
The F7's weren't intentional, I'll check now if it's valid and filter it out if not.

Reply:
These values are all defined in the sources/headers files for those specific chips (which was the intentional source for the L1 series), but I can't find a reference to it anywhere in the reference manuals.
I can try and strip it out if necessary but it seems fairly legitimate.